src/languages/arcade.js

```diff
@@ -1,7 +1,7 @@
 /*
  Language: ArcGIS Arcade
  Category: scripting
- Author: John Foster <jfoster@esri.com>
+ Author: John Foster <jfoster@esri.com>, Kristian Ekenes <kekenes@esri.com>
```

Comment:
```suggestion
```

We're no longer tracking grammar authors/contributors this way, instead we credit all contributors by linkinh to Github.